### ISSUE TITLE
[feature request] Can we have a "full" search page, it is useful when we don't know the keyword of search, then we can use filters first 

### ISSUE THREAD

**Issue by mobilelifeful** [hugomods/search#164]
For example, this is the pop up after we click the search
![image](https://github.com/hugomods/search/assets/91811647/f706e962-9bd1-428e-9d57-5bc8a3a7c6d8)

if we can have a full search page (not like the current pop up), where all the filters are expanded. Similar like below, so people can filter first with clicks of choices.   I think there should be no differences with what current search can do, but a dedicated full search page (not as pop up) Thank you.
![image](https://github.com/hugomods/search/assets/91811647/ae194de6-158c-4880-ae67-59a9e99bdbc9)

**Comment by razonyang** [comment]
Do you mean the [HB theme](https://github.com/hbstack/theme)?

**Comment by razonyang** [comment]
> where all the filters are expanded.

No, it doesn't make sence for me, if the users don't know what to search then they probably not want to search.

**Comment by mobilelifeful** [comment]
> > where all the filters are expanded.
> 
> No, it doesn't make sence for me, if the users don't know what to search then they probably not need to search.

Yes, agree on that. My reasoning is weak. Currently, the search pop up is nice and does search well. 

Maybe, it is just me (or someone who are not the authors of the hbstack know the theme inside out), in the visitors perspective, we find a nice site and would like have a full search page for them to poke around to look for content, where victors are not sure what the site have, but still use the filters to find if something they like.

**Comment by razonyang** [comment]
> full search page for them to poke around to look for content

It sounds overweight of search, since there are toxonimies (categories, tags and so on) list for this, the only advantage is able to combine multiple filters to search pages. If I understand correctlly, you want to search with filters only (without keywords)?

> The full search page will have different UI, not like the current pop up. We can have filters all expanded like the side bar to the left.

It maybe good to have, but I've no plan on this for now.

**Comment by razonyang** [comment]
Now this module allow searching without keywords since `v0.9.0`, not sure if this suit your needs, please file another issue if it doesn't.

/CC @techhubhub
